extract_thumbnail_from_video.py

```diff
@@ -0,0 +1,39 @@
+import cv2
```

Comment:
Add a README.md, with screenshots of the workings of the projects.
If possible, add video of the same for the projects.
README.md should explain everything about the project.

Reply:
Sorry for the delay. README.md has been added.

Reply:
@NitkarshChourasia please review again